app/controllers/user_blocks_controller.rb

```diff
@@ -24,7 +24,7 @@ def create
       @user_block.blocked.stop_following(current_user)
       render json: { result: "blocked" }
     else
-      render json: { result: "error: #{@user_block.errors.full_messages}" }
+      render json: { error: @user_block.errors.full_messages.join(", ") }, status: :unprocessable_entity
```

Comment:
I'm surprised nothing had to change in the JS 👀

Reply:
I changed some JS to match this. 😄